sermant-plugins/sermant-service-registry/spring-cloud-registry-service/src/main/java/io/sermant/registry/service/impl/GraceServiceImpl.java

```diff
@@ -83,8 +86,14 @@ private void notifyToGraceHttpServer(String address, Map<String, Collection<Stri
     }
 
     private void execute(String address, Map<String, Collection<String>> header) {
-        HttpClientUtils.INSTANCE.doPost(GRACE_HTTP_SERVER_PROTOCOL + address + GraceConstants.GRACE_NOTIFY_URL_PATH,
-                REQUEST_BODY, header);
+        for (int i = 0; i < RETRY_TIME; i++) {
+            HttpClientResult result = HttpClientUtils.INSTANCE.doPost(
+                    GRACE_HTTP_SERVER_PROTOCOL + address + GraceConstants.GRACE_NOTIFY_URL_PATH,
+                    REQUEST_BODY, header);
+            if (result.getCode() == GraceConstants.GRACE_HTTP_SUCCESS_CODE) {
+                break;
+            }
+        }
```

Comment:
If the notification keeps failing, it is recommended to add a log

Reply:
done